code/game/objects/items/miscellaneous.dm

```diff
@@ -337,6 +337,28 @@
 	to_chat(M, msg)
 	new /obj/effect/pod_landingzone(get_turf(src), pod)
 
+/obj/item/choice_beacon/petchoice
+	name = "Multiple choice animal delivery beacon"
+	desc = "There are no faster ways, only more humane."
+	var/default_name = "Petsie"
+	
+/obj/item/choice_beacon/petchoice/generate_display_names()
+	var/static/list/pet
+	if(!pet)
+		pet = list()
+		var/list/templist = list(/mob/living/simple_animal/pet/cat,
```

Comment:
This is weird, why are you mixing pets with beacons?

Reply:
its a spawn beacon not a random pet spawn.

Reply:
but yeah maybe i should name it better...